app/(candidates)/candidates/candidate-dashboard/my-page/ContentBlocksEditor.tsx

```
@@ -0,0 +1,525 @@
+            `Upload failed: ${
+              error instanceof Error ? error.message : String(error)
+            }`
+          );
+        } finally {
+          setUploading(block.order, false);
+          setProgress(block.order, 0);
+        }
+      };
+
+      inner = (
+        <>
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)